tests/functional_tests/iterator_tests/test_column_stencil.py

```diff
@@ -254,4 +254,82 @@ def test_kdoublesum_scan(program_processor, lift_mode, kstart, reference):
             assert np.allclose(reference[n], np.asarray(out)[n])
 
 
+@fundef
+def sum_shifted(inp0, inp1):
+    return deref(inp0) + deref(shift(K, 1)(inp1))
+
+
+@fendef(column_axis=KDim)
+def sum_shifted_fencil(out, inp0, inp1, k_size):
+    closure(
+        cartesian_domain(named_range(KDim, 0, k_size)),
```

Comment:
```suggestion
        cartesian_domain(named_range(KDim, 1, k_size)),
```
Just to ensure the start index is taken into account correctly.

Reply:
done